src/components/DashboardLayout/LeftNav.tsx

```diff
@@ -54,12 +57,21 @@ export default function LeftNav({ className }: { className?: string }) {
             </div>
           </Dialog>
         </Transition>
-        <InstanceList className="pt-12 pb-4">
+        <InstanceNestedBarStates className="mt-6"></InstanceNestedBarStates>
+        <InstanceList className="mt-6">
           <div className="items-begin mb-4 flex w-full flex-row items-center justify-center gap-4">
             <Button
-              label="Add"
-              className="w-fit"
-              icon={faPlus}
+              label="New instance..."
```

Comment:
<img width="143" alt="image" src="https://user-images.githubusercontent.com/44907675/218777764-95efb297-8f60-4dcb-8ce3-0e5d51e99aaa.png">
This button has a weird double border on `focus-visible`